consul/base.py

```diff
@@ -138,21 +138,33 @@ def callback(
         is_json=False,
         index=False,
         one=False,
-        allow_404=True):
+        allow_404=True,
+        decode=None,
+        is_id=False):
```

Comment:
`is_200` and `is_json` are mutually exclusive options. I wonder if there's a way to structure the function signature to make this obvious...